app/components/Documentation/EditDocumentation.tsx

```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
     });
   };
 
+  const onClickArticle = (id: string): void => {
+    if (
+      typeof setViewArticleDispatch === 'function' &&
+      typeof setEditArticleDispatch === 'function'
+    ) {
+      setViewArticleDispatch(id);
+      setEditArticleDispatch({
+        id: id,
+        parentId: '1',
+        isEditArticle: false,
+      });
+    }
+  };
+  const onClickPrevArticle = (): void => {
+    const prevArrayId = treeArray.filter(
+      (item) => item?.treeIndex == currentArrayIndex - 1,
+    )[0]?.node.id;
+    onClickArticle(prevArrayId);
+  };
+
+  const onClickNextArticle = (): void => {
+    const nextArrayId = treeArray.filter(
+      (item) => item?.treeIndex == currentArrayIndex + 1,
+    )[0]?.node.id;
+    onClickArticle(nextArrayId);
+  };
+
+  const onClickNextTitle = (): string => {
```

Comment:
combine functions onClickNextTitle and onClickPrevTitle into one, use a closure

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9